ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -135,6 +135,13 @@ oc wait --for=condition=Updated mcp/worker --timeout=30m
 
 }
 
+# inject porxy information into haproxy.service if deploying with proxy configuration
+proxy_info=$(cat ${SHARED_DIR}/install-config.yaml | grep -oP 'httpProxy\s*:\s*\K.*')
+if [ "x${proxy_info}" != "x" ]; then
+  echo "$(date -u --rfc-3339=seconds) - inject proxy env into haproxy service..."
```

Comment:
```suggestion
  echo "inject proxy env into haproxy service..."
```
nitpick: no need for a date as its being printed in build log already:
>INFO[2021-11-09T07:45:56Z] 2021-11-09 07:45:52+00:00 - sourcing context from vsphere_context.sh...

but I'm okay with leaving it if it has some other purpose